lib/dentaku/evaluator.rb

```diff
@@ -162,5 +162,20 @@ def round_int(*args)
     def not(*args)
       Token.new(:logical, ! evaluate_token_stream(args[2..-2]).value)
     end
+
+    (Math.methods - Object.methods).each do |method|
+      define_method method do |*args|
+        function, _, *tokens, _ = *args
+        values = tokens.map(&:value)
+        result = if Math.method(method).arity == 2
```

Comment:
It might be a bit cleaner to filter out any grouping tokens and then just assert that values.length == arity and just call `Math.send(method *values)`